Makefile

```diff
@@ -1,26 +1,68 @@
+PROGRAM = Compile
+VERSION = git-$(shell date +%Y%m%d)
+GOBO_PROGRAMS ?= /Programs
```

Comment:
The variable was called `goboPrograms` on purpose, so that it auto-inherits the right thing when running on Rootless (which we might get to support again in the future — it kind of still works; I think Lucas uses it)